docs/user/save_load_resume_exp.md

```diff
@@ -0,0 +1,309 @@
+policy/Entropy                                0.508267
+policy/KL                                     0.00908262
+policy/KLBefore                               0
+policy/LossAfter                             -0.00815415
+policy/LossBefore                            -1.2031e-07
+policy/Perplexity                             1.66241
+policy/dLoss                                  0.00815403
+---------------------------------------  ---------------
+2020-06-26 13:42:18 | epoch #19 | Obtaining samples for iteration 19...
+2020-06-26 13:42:19 | epoch #19 | Logging diagnostics...
+2020-06-26 13:42:19 | epoch #19 | Optimizing policy...
+2020-06-26 13:42:19 | epoch #19 | Computing loss before
+2020-06-26 13:42:19 | epoch #19 | Computing KL before
+2020-06-26 13:42:19 | epoch #19 | Optimizing
+2020-06-26 13:42:19 | epoch #19 | Start CG optimization:
+#parameters: 1282, #inputs: 52, #subsample_inputs: 52
+2020-06-26 13:42:19 | epoch #19 | computing loss before
+2020-06-26 13:42:19 | epoch #19 | computing gradient
+2020-06-26 13:42:19 | epoch #19 | gradient computed
+2020-06-26 13:42:19 | epoch #19 | computing descent direction
+2020-06-26 13:42:20 | epoch #19 | descent direction computed
+2020-06-26 13:42:20 | epoch #19 | backtrack iters: 4
+2020-06-26 13:42:20 | epoch #19 | optimization finished
+2020-06-26 13:42:20 | epoch #19 | Computing KL after
+2020-06-26 13:42:20 | epoch #19 | Computing loss after
+2020-06-26 13:42:20 | epoch #19 | Fitting baseline...
+2020-06-26 13:42:20 | epoch #19 | Saving snapshot...
+2020-06-26 13:42:20 | epoch #19 | Saved
+2020-06-26 13:42:20 | epoch #19 | Time 19.25 s
+2020-06-26 13:42:20 | epoch #19 | EpochTime 1.96 s
+---------------------------------------  ---------------
+EnvExecTime                                   0.286301
+Evaluation/AverageDiscountedReturn           86.602
+Evaluation/AverageReturn                    200
+Evaluation/CompletionRate                     0
+Evaluation/Iteration                         19
+Evaluation/MaxReturn                        200
+Evaluation/MinReturn                        200
+Evaluation/NumTrajs                          52
+Evaluation/StdReturn                          0
+Extras/EpisodeRewardMean                    200
+LinearFeatureBaseline/ExplainedVariance       0.999973
+PolicyExecTime                                1.24483
+ProcessExecTime                               0.076405
+TotalEnvSteps                            204058
+policy/Entropy                                0.529518
+policy/KL                                     0.00595805
+policy/KLBefore                               0
+policy/LossAfter                             -0.00285657
+policy/LossBefore                            -2.5089e-07
+policy/Perplexity                             1.69811
+policy/dLoss                                  0.00285632
+---------------------------------------  ---------------
+```
+
+You may find the full example at `examples/jupyter/pre-trained_trpo_gym_tf_cartpole.ipynb`.
```

Comment:
Can you point to a non-Jupyter example? We don't really maintain this very well right now.

Reply:
Removed `examples/jupyter/pre-trained_trpo_gym_tf_cartpole.ipynb`. Instead, add example `examples/tf/trpo_gym_tf_cartpole_pretrained.py`